api/service/explorer/service.go

```diff
@@ -247,6 +251,15 @@ func (s *Service) GetExplorerAddress(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(data.Address)
 }
 
+// GetExplorerNodes serves /nodes end-point.
+func (s *Service) GetExplorerNodes(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	var nodes int
+	nodes = 50 + rand.Int()%50 // TODO(ricl): get nodes count.
```

Comment:
get the length of consensus.PublicKeys.  You may add a variable in internal/configs/node/config.go, ConfigType, and update it regularly with the start of each consensus.